src/trace.rs

```diff
@@ -295,21 +295,37 @@ impl From<&Event> for EventData {
 
 impl From<&Event> for MessageData {
     fn from(event: &Event) -> MessageData {
+        let mut props = event
+            .attributes
+            .iter()
+            .map(|kv| (kv.key.as_str().into(), (&kv.value).into()))
+            .collect::<Properties>();
+
+        let log_level = severity_level(props.remove(&"level".into()));
```

Comment:
Can you document that `level` comes from the tracing crate?

Reply:
I think the name "level" comes from here: https://github.com/tokio-rs/tracing/blob/a0126b2e2d465e8e6d514acdf128fcef5b863d27/tracing-opentelemetry/src/subscriber.rs#L839. Might be nice to put that link in comment here.